### PR TITLE
#7602 | Fix incorrect attribute

### PR DESCRIPTION
**Pull request by rottina**
What's changed, or what was fixed?
- adding anonymous attribute as per the spec: https://www.w3.org/TR/html5/infrastructure.html#cors-settings-attributes
- Fixing unicode colons that the tests were warning about

**Fixes:** #7602

**Target Live Date:** YYYY-MM-DD

- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `npm test` locally and all tests pass.
- [ ] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Comment by petele** [comment]
@jakearchibald & @surma - can one of you take a look at this and approve/merge if it's correct?

File: src/content/en/fundamentals/performance/resource-prioritization.md [hunk 1]
```diff
@@ -157,7 +157,7 @@ of your site, as well as avoid jarring flashes between system fonts and your
 preferred ones, you can use `<link rel="preload">` in your HTML to let the
 browser know immediately that a font is needed.
 
-    <link rel="preload" as="font" crossorigin="crossorigin" type="font/woff2" href="myfont.woff2">
+    <link rel="preload" as="font" crossorigin="anonymous" type="font/woff2" href="myfont.woff2">
```
Comment (jakearchibald) [inline src/content/en/fundamentals/performance/resource-prioritization.md]:
```suggestion
    <link rel="preload" as="font" crossorigin type="font/woff2" href="myfont.woff2">
```

File: src/content/es/fundamentals/performance/resource-prioritization.md [hunk 2]
```diff
@@ -157,7 +157,7 @@ de tu sitio, así como para evitar cambios molestos entre las fuentes del sistem
 que configuraste para el sitio, puedes usar `<link rel="preload">` en tu código HTML con el objetivo de informar de inmediato al
 navegador que se necesita una fuente.
 
-    <link rel="preload" as="font" crossorigin="crossorigin" type="font/woff2" href="myfont.woff2">
+    <link rel="preload" as="font" crossorigin="anonymous" type="font/woff2" href="myfont.woff2">
```
Comment (jakearchibald) [inline src/content/es/fundamentals/performance/resource-prioritization.md]:
```suggestion
    <link rel="preload" as="font" crossorigin type="font/woff2" href="myfont.woff2">
```

File: src/content/id/fundamentals/performance/resource-prioritization.md [hunk 3]
```diff
@@ -157,7 +157,7 @@ situs Anda, serta menghindari konflik antara font sistem dan
 font yang dipilih, Anda dapat menggunakan `<link rel="preload">` di HTML agar
 browser segera mengetahui bahwa font diperlukan.
 
-    <link rel="preload" as="font" crossorigin="crossorigin" type="font/woff2" href="myfont.woff2">
+    <link rel="preload" as="font" crossorigin="anonymous" type="font/woff2" href="myfont.woff2">
```
Comment (jakearchibald) [inline src/content/id/fundamentals/performance/resource-prioritization.md]:
```suggestion
    <link rel="preload" as="font" crossorigin type="font/woff2" href="myfont.woff2">
```

File: src/content/it/fundamentals/performance/resource-prioritization.md [hunk 4]
```diff
@@ -165,7 +165,7 @@ testuale del tuo sito e per evitare fastidiosi flash tra i font di sistema e
 quelli preferiti puoi usare `<link rel="preload">` nel tuo HTML per far sapere
 immediatamente al browser che è necessario un font.
 
-    <link rel="preload" as="font" crossorigin="crossorigin" type="font/woff2" href="myfont.woff2">
+    <link rel="preload" as="font" crossorigin="anonymous" type="font/woff2" href="myfont.woff2">
```
Comment (jakearchibald) [inline src/content/it/fundamentals/performance/resource-prioritization.md]:
```suggestion
    <link rel="preload" as="font" crossorigin type="font/woff2" href="myfont.woff2">
```

File: src/content/ja/fundamentals/performance/resource-prioritization.md [hunk 5]
```diff
@@ -147,7 +147,7 @@ as 属性以外の部分はおそらく予想どおりでしょう。
 
 
 
-    <link rel="preload" as="font" crossorigin="crossorigin" type="font/woff2" href="myfont.woff2">
+    <link rel="preload" as="font" crossorigin="anonymous" type="font/woff2" href="myfont.woff2">
```
Comment (jakearchibald) [inline src/content/ja/fundamentals/performance/resource-prioritization.md]:
```suggestion
    <link rel="preload" as="font" crossorigin type="font/woff2" href="myfont.woff2">
```

File: src/content/ko/fundamentals/performance/resource-prioritization.md [hunk 6]
```diff
@@ -157,7 +157,7 @@ Chrome의 현재 우선순위 상태에 대해 자세히 설명하고 있습니
 `<link rel="preload">`를 HTML에 사용하면
 글꼴이 필요하다는 것을 브라우저가 즉시 알 수 있습니다.
 
-    <link rel="preload" as="font" crossorigin="crossorigin" type="font/woff2" href="myfont.woff2">
+    <link rel="preload" as="font" crossorigin="anonymous" type="font/woff2" href="myfont.woff2">
```
Comment (jakearchibald) [inline src/content/ko/fundamentals/performance/resource-prioritization.md]:
```suggestion
    <link rel="preload" as="font" crossorigin type="font/woff2" href="myfont.woff2">
```

File: src/content/pt-br/fundamentals/performance/resource-prioritization.md [hunk 7]
```diff
@@ -157,7 +157,7 @@ do seu site e também evitar dissonâncias desagradáveis entre as fontes do sis
 favoritas, você pode usar o `<link rel="preload">` no seu HTML. Dessa forma, o
 navegador entenderá imediatamente que uma fonte é necessária.
 
-    <link rel="preload" as="font" crossorigin="crossorigin" type="font/woff2" href="myfont.woff2">
+    <link rel="preload" as="font" crossorigin="anonymous" type="font/woff2" href="myfont.woff2">
```
Comment (jakearchibald) [inline src/content/pt-br/fundamentals/performance/resource-prioritization.md]:
```suggestion
    <link rel="preload" as="font" crossorigin type="font/woff2" href="myfont.woff2">
```

File: src/content/zh-cn/fundamentals/performance/resource-prioritization.md [hunk 8]
```diff
@@ -148,7 +148,7 @@ book_path: /web/fundamentals/_book.yaml
 
 
 
-    <link rel="preload" as="font" crossorigin="crossorigin" type="font/woff2" href="myfont.woff2">
+    <link rel="preload" as="font" crossorigin="anonymous" type="font/woff2" href="myfont.woff2">
```
Comment (jakearchibald) [inline src/content/zh-cn/fundamentals/performance/resource-prioritization.md]:
```suggestion
    <link rel="preload" as="font" crossorigin type="font/woff2" href="myfont.woff2">
```

**Comment by jakearchibald** [comment]
Don't need the `="anonymous"` bit, as it's the default value. Other than that it's a change worth making!

I can't vouch for the char changes in zh-cn though.

**Pull request merged**